lib/template/lib/routes.rb

```diff
@@ -4,7 +4,7 @@
   use Pliny::Middleware::CORS
   use Pliny::Middleware::RequestID
   use Pliny::Middleware::RequestStore, store: Pliny::RequestStore
-  use Pliny::Middleware::Timeout, timeout: Config.timeout if Config.timeout > 0
+  use Rack::Timeout if Config.timeout > 0
```

Comment:
@pedro I don't think that `Routes` is inside the `Pliny` namespace (although it probably was at some point) so I think that this will actually grab the obsolete Ruby `Config` module (now `RbConfig`) instead of `Pliny::Config`, which was probably what was intended. I see now that it was wrong before too though, doh.